src/modules/wallet/utils/helpers.js

```diff
@@ -0,0 +1,2 @@
+export const calculateSupply = (balance, totalSupply) =>
```

Comment:
If its not reused can we move this the place where its only being used?

Reply:
Fixed